cmd/snap/cmd_snapshot.go

```diff
@@ -469,3 +488,28 @@ func (x *exportSnapshotCmd) Execute([]string) (err error) {
 
 	return nil
 }
+
+type importSnapshotCmd struct {
+	clientMixin
+	durationMixin
+	Positional struct {
+		Filename string `long:"filename"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+func (x *importSnapshotCmd) Execute([]string) error {
+	filename := x.Positional.Filename
+	f, err := os.Open(filename)
+	if err != nil {
+		return fmt.Errorf("error accessing file: %v", err)
+	}
+	defer f.Close()
+
+	importSet, err := x.client.SnapshotImport(f)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(Stdout, "Imported snapshot with %d snaps as snapshot ID %d\n", len(importSet.Snaps), importSet.ID)
```

Comment:
this should probably list the snaps names no, instead of just the count?

Reply:
This is now using what "snapshot save" does